projects/github-actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -76,22 +76,33 @@ async function getMilestoneDates( plugin, nextMilestone ) {
 		// Spaces between words.
 		.join( ' ' );
 
+	let pluginMessage = '';
+	if ( ! releaseDate ) {
+		pluginMessage = `No scheduled milestone found for this plugin.`;
+	} else if ( plugin === 'jetpack' ) {
+		pluginMessage = `The Jetpack plugin has different release cadences depending on the platform:
+
+- WordPress.com Simple releases happen semi-continuously (PCYsg-Jjm-p2).
+- WoA releases happen weekly.
+- Releases to self-hosted sites happen monthly:
+    - Scheduled release: _${ releaseDate }_
+`;
+		if ( codeFreezeDate ) {
+			pluginMessage += `    - Code freeze: _${ codeFreezeDate }_\n`;
+		}
```

Comment:
What do you think about simplifying things a bit?

```suggestion
    - Scheduled release: _${ releaseDate }_
${ codeFreezeDate ? `    - Code freeze: _${ codeFreezeDate }_` : '' }
`;
```

Reply:
Personally I find this a complification rather than a simplification. :sweat_smile: